roles/create_bastion_hypershift/tasks/main.yaml

```
@@ -0,0 +1,51 @@
+---
+- name: Get ssh key of local host
+  slurp:
+    src: "~/.ssh/{{ env.ansible_key_name }}.pub"
+  register: encoded_ssh
+  delegate_to: localhost
+
+- name: Decode the ssh key
+  set_fact:
+    ssh_key: "{{encoded_ssh.content | b64decode |replace('\n','')}}"
+
+- name: Create Directory for images and bastion.ks
+  shell: mkdir -p /home/libvirt/images/
```

```suggestion
- name: Create Directory for images and bastion.ks
  file: 
    path: /home/libvirt/images/
    recurse: true
    state: directory
```
Please use Ansible modules wherever possible, shell module is not idempotent.

This is untested ^